exchanges/anx/anx_test.go

```diff
@@ -227,6 +228,19 @@ func TestFormatWithdrawPermissions(t *testing.T) {
 	}
 }
 
+func TestGetDataToken(t *testing.T) {
+	a.SetDefaults()
+	TestSetup(t)
+
+	response, err := a.GetDataToken()
```

Comment:
Slight nit, can just bunch response, err := a.GetDataToken() and check for err straight under it without a space.

Reply:
I like the space, but since I've received other feedback on this, I'll do err checking on the line below 👍